### PR TITLE
chore: update Roboto Mono and Proxima Nova files to woff2

### PR DESCRIPTION
**Pull request by ChitlangeSahas**
Closes https://github.com/influxdata/clockface/issues/771

### Changes

I converted the `otf` fonts to `woff2` formats using https://cloudconvert.com/otf-to-woff2  so that they are easily usable in the UI.
### Screenshots

Screen Shot 2022-05-21 at 2 24 24 PM

### Checklist
Check all that apply

- [x] Updated documentation to reflect changes
- [x] Added entry to top of Changelog with link to PR (not issue)
- [x] Tests pass
- [ ] Peer reviewed and approved
- [x] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

**Review by TCL735 — APPROVED** [review]

**Comment by TCL735** [comment]
💯

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 4.2.1 (2022-05-21)
```
Comment (hoorayimhelping) [inline CHANGELOG.md:3]: let's update this to today since it didn't get merged in on Friday

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+
+- [769](https://github.com/influxdata/clockface/pull/769): Added optional property to TreeNavSubMenu
```
Comment (hoorayimhelping) [inline CHANGELOG.md:5]: >Added optional property to TreeNavSubMenu

lies.

**Review by hoorayimhelping — COMMENTED** [review]
docs: make sure the changelog is correct

**Pull request merged**